src/app/components/Menu/MenuList.tsx

```diff
@@ -18,11 +20,13 @@ function List({ position = "left", children }: Props) {
       leaveTo="opacity-0 scale-95"
     >
       <Menu.Items
-        className={`${
+        className={classNames(
           position === "left"
             ? "left-0 origin-top-left"
-            : "right-0 origin-top-right"
-        } absolute z-50 mt-2 py-1 w-56 rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 focus:outline-none dark:bg-surface-12dp dark:border-0`}
+            : "right-0 origin-top-right",
+          fullWidth ? "w-full" : "w-56",
```

Comment:
Why do we need a width here?

Reply:
@reneaaron I think to make the menu a consistent width no matter where it is used, and make sure it does not automatically expand if one of the items is wide. I am not sure I understand the question, does this answer it?